package.json

```diff
@@ -164,6 +164,7 @@
     "abacus-publish": "file:tools/publish",
     "abacus-replace-template": "file:tools/replace-template",
     "abacus-secured-itest": "file:test/secured",
+    "abacus-services-hours-scenario-test": "file:test/scenario/metering-plans/standard-services-hours",
```

Comment:
Maybe `metering-plans` could be a module instead and some time in the future it would contain tests for all metering plans. In such a way, a lot of the boilerplate code can be reused.

Otherwise, once we decide to implement a new plan for applications, it would need to have its own scenario test module.